app/views/dashboard/confirmDeleteDraftCase.scala.html

```diff
@@ -0,0 +1,69 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.play.views.html.helpers.form
+@import utils.Formatters._
+@import actions.AuthenticatedRequest
+@import config.ApplicationConfig
+@import config.ApplicationConfig._
+@import helper._
+
+@(model: models.DraftCase)(implicit request: AuthenticatedRequest[_], messages: Messages, config: ApplicationConfig)
+
+ @main_template(
+  title = Messages("draftCases.confirm.delete.title"),
+  individualId = Some(request.personId),
+  contentHeaderText = Some(Messages("draftCases.confirm.delete.title")),
+  contentHeaderHidden = false,
+  topNavigation = false
+ ) {
+
+
+  <h2 class="heading-medium">@Messages("draftCases.confirm.delete.message.1")</h2>
+
+   @form(routes.ManageDrafts.confirmDelete(model.id)) {
+       <table id="draftsTable" class="margin-top-20 margin-bottom-20">
+        <caption class="visuallyhidden">@Messages("draftCases.confirm.table.caption")</caption>
+        <colgroup>
+         <col width="60%">
+         <col width="20%">
```

Comment:
Is there a reason for the table stuff here? is it not just CSS we need to get right?

Reply:
resolved